packages/trader/src/Modules/SmartChart/Components/top-widgets.tsx

```diff
@@ -52,11 +51,19 @@ const TopWidgets = ({
         document.getElementById('app_contents') as Element | DocumentFragment
     );
 
+    if (is_mobile) {
+        return (
+            <React.Fragment>
+                {InfoBox}
+                {portal}
+            </React.Fragment>
+        );
+    }
+
     return (
         <React.Fragment>
             {InfoBox}
-            <MobileWrapper>{portal}</MobileWrapper>
-            <DesktopWrapper>{ChartTitleLocal}</DesktopWrapper>
+            {ChartTitleLocal}
```

Comment:
instead of if-return construction, we can do smth like this:
```
return (
    <React.Fragment>
      {InfoBox}
      {is_mobile ? portal :ChartTitleLocal }
    <React.Fragment/>
)
```